### DISCUSSION TITLE
Not compatible with v0.134.0?

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#495] [Q&A]
Hi @razonyang !

[This](https://de63c53c.pushevs.pages.dev/) how my website looks if I update it to the latest Hugo version (v0.134.0).

**Comment by razonyang** [comment]
There may be some changes of v0.134.0, will review later. 

Btw, I'd recommended using the fixed Hugo version for production sites, to get stable builds.

**Comment by razonyang** [comment]
Fixed since [github.com/hbstack/blog@v0.37.4](https://github.com/hbstack/blog/releases/tag/v0.37.4).
